analyzer-comments/elixir/solution/unless_with_else.md

```diff
@@ -0,0 +1,7 @@
+ # unless having else block
+ 
+ Unless conditions should avoid having an `else` block.
+ 
+ The reason for this is not a technical but a human one. The `else` in this
+ case will be executed when the condition is met, which is the opposite of
+ what the wording seems to apply.
```

Comment:
I see a few small grammar problems here (also formatting - no need for the leading single space on every one)

- It's not the condition that needs to avoid "else", but the programmer.
- The verb "apply" doesn't fit in this context.

Here's my suggestion for a small rewording, already with @jiegillet suggestion applied (to mention `if`).

```suggestion
# unless having else block

Avoid using `unless` expressions with an `else` block. Use an `if` expression instead.

The reason for this is not a technical but a human one. The `else` in this
case will be executed when the condition is met, which is the opposite of
what the wording suggests.
```